_posts/components/2018-10-01-autocomplete.html

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title:  Autocomplete
+category: components
+description: Whenever a field have the ability of autocomplete its value we should add a hint in form of placeholder and 
+             also an icon on the left that shows a spinner while loading from remote
+---
+
+%h5 Input with autocompletion
+
+.form-group#form-group-id
+    %label{ for: 'input-id' }
+        Label:
+    .input-group
+        .input-group-prepend
+            %span.input-group-text#search-icon-id
+                %i.fa.fa-search
+                %i.fas.fa-spinner.fa-spin.d-none
+        %input.form-control#input-id{type: 'text', placeholder: 'Type to autocomplete...'}
```

Comment:
Missing spaces inside the brackets (so it should be like line 12). This would trigger Rubocop in OBS.